src/web/crate_details.rs

```diff
@@ -145,11 +146,15 @@ impl CrateDetails {
                 .collect()
         };
 
+        let repository_url: Option<String> = krate.get("repository_url");
         let github_metadata = if krate.get::<_, Option<String>>("github_repo").is_some() {
             Some(GitHubMetadata {
                 issues: krate.get("github_issues"),
                 stars: krate.get("github_stars"),
                 forks: krate.get("github_forks"),
+                name: repository_url
+                    .as_ref()
+                    .and_then(|u| u.splitn(2, "/github.com/").nth(1).map(|s| s.to_owned())),
```

Comment:
The up to date repository name is already present in the `github_repos` table, we should fetch it from there.

Reply:
Indeed, thanks!